agent/containers/images/README.md

```diff
@@ -3,7 +3,7 @@
 Container image building requires both the `buildah` and `jinja2` CLI
 commands to be present.
 
-On Fedora 32 and later systems, use the following command to install the
+On Fedora 33 and later systems, use the following command to install the
```

Comment:
What about on Centos (and/or RHEL)?

Reply:
What does the change from 32 to 33 mean for CentOS or RHEL?

Reply:
The README discusses a number of points regarding Fedora.  At one spot, it also mentions CentOS.  It seems odd that it mentions CentOS there without calling it out in the other places that it mentions details for Fedora.  (I didn't notice _any_ references to RHEL, which did seem slightly odd.)

It seems worthwhile, while we were updating this document, for us to give it a full once-over, to make sure that it still says what we want it to say....

Reply:
Tracked in #2379.